auto_py_to_exe/utils.py

```diff
@@ -39,10 +40,37 @@ def open_output_in_explorer(output_directory, input_filename, is_one_file):
         return False
     return True
 
+def check_latest_pyinstaller_version():
+    """Check the latest version of PyInstaller"""
+    try:
+        response = requests.get("https://api.github.com/repos/pyinstaller/pyinstaller/releases/latest")
+        response.raise_for_status()
+        latest_version = response.json()["tag_name"]
+        if latest_version.startswith("v"):
+            latest_version = latest_version[1:]
+        return latest_version
+    except requests.exceptions.RequestException:
+        return None
 
 def get_warnings():
     warnings = []
 
+    # Check pyinstaller version is it latest
+    try:
+        latest_version = check_latest_pyinstaller_version()
+        if latest_version is None:
+            raise Exception("Unable to check for the latest version of PyInstaller.")
+        elif latest_version != pyinstaller_version_string:
+            message = "You are using an outdated version of PyInstaller."
+            message += f"\nLatest version: {latest_version}"
+            message += f"\nYour version: {pyinstaller_version_string}"
+            message += "\nPlease upgrade PyInstaller: python -m pip install pyinstaller --upgrade"
```

Comment:
Updating would be an optional move, maybe this wording is a bit more friendly?

```suggestion
            message = "A newer version of PyInstaller has been released."
            message += f"\nLatest version: {latest_version}"
            message += f"\nYour version: {pyinstaller_version_string}"
            message += "\nUpgrade using: python -m pip install pyinstaller --upgrade"
```